### PR TITLE
[CMS-1096] De-emphasize terminus 1.x and 2.x docs.

### PR DESCRIPTION
**Pull request by kporras07**
Closes CMS-1096

## Summary

Several pages changed, mostly under:
**[Terminus guide](https://pantheon.io/docs/terminus)** - De-emphasizes and remove terminus 1.x and 2.x references in docs.

**Release**:
- [X] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `https://pantheon.io/docs/terminus-2-0` => `https://pantheon.io/docs/terminus/terminus-3-0` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/terminus-2-0.md [hunk 1]
```diff
@@ -6,20 +6,14 @@ categories: [platform]
 tags: [cli, drush, local, wp-cli, terminus]
 ---
 
-<Alert title="Note" type="info" >
+<Alert title="Warning" type="danger" >
```
Comment (kporras07) [inline source/content/terminus-2-0.md:9]: Actually, I'd love to remove this page and redirect to terminus-3-0 (https://pantheon.io/docs/terminus/terminus-3-0). Let me know if this is possible and how could I help to make it happen

Comment (whitneymeredith) [reply source/content/terminus-2-0.md:9]: Agreed!

**Review by whitneymeredith — APPROVED** [review]

**Comment by whitneymeredith** [comment]
@kporras07 LGTM. I'll wait for the @pantheon-systems/cms-ecosystem to sign off on this.

File: source/content/terminus/07-create.md [hunk 2]
```diff
@@ -195,49 +112,20 @@ You must complete the steps below if you want to share your plugin with others.
...
-Terminus 1 and 2 do not load a plugin's external libraries until immediately before one of its commands is executed to avoid conflicts between the dependencies of different plugins. That means that you cannot use any external classes in your plugin's constructor. Terminus 3 does not have this restriction.
-
-</Alert>
 
 ## Coding Standards
 
 Pantheon recommends adopting Terminus core standards if you plan to distribute your plugin and/or add it to an open source license and encourage contributions. Some basic principles to follow are:
 
-- Ensure compatibility with PHP 5.5, 5.6 and 7
+- Ensure compatibility with PHP >=7.8 and 8
```
Comment (namespacebrian) [inline source/content/terminus/07-create.md:128]: Should this be 7.4? 😉

File: source/content/supported-terminus.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
----
-title: Supported Terminus Versions
```
[the remark this answers is not in the document]
Comment (kporras07) [reply source/content/supported-terminus.md:2]: A redirect should definitely be created. @whitneymeredith are you taking care of this after merging?

File: source/content/terminus/07-create.md (hunk 2 above)
Comment (kporras07) [reply source/content/terminus/07-create.md:128]: Great catch! Thanks!

File: source/content/supported-terminus.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/supported-terminus.md:2]: That's our typical redirect workflow.

**Review by namespacebrian — APPROVED** [review]

**Comment by kporras07** [comment]
@whitneymeredith all yours. It has now been signed off by cms eco team :)

**Pull request merged**